RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,16 @@
+
+ 1. [ ] (Create release branch)
+ 1. [ ] With each 20XX.1 release, update Copyright notices accordingly
+ 1. [ ] Check wheel install on debian stretch produces an OK error message due to too old python
+ 1. [ ] Write Release Notes
+ 1. [ ] Update Readme
+ 1. [ ] Update Copyright notice if year roll over
+ 1. [ ] Tag commit
+ 1. [ ] Push tag
+ 1. [ ] bump/create demo docker
+ 1. [ ] Wait for CI build to finish (this will upload sdist+wheels to pypi, conda builds to conda cloud)
```

Comment:
Zenodo release

Is the github release automatic?

Reply:
It's the other way round.